app/controllers/application_forms_controller.rb

```diff
@@ -9,7 +9,7 @@ def create
     begin
       @application_form.save!
       @user = @application_form.user
-      save_user_to_session!
+      save_user_to_session! unless SessionService.is_signed_in?(session)
```

Comment:
would it be worth covering this case with a spec that makes multiple form submissions within one session (which is a quite realistic scenario)?